src/main/java/com/dongtronic/diabot/commands/NightscoutCommand.kt

```diff
@@ -113,6 +115,39 @@ class NightscoutCommand(category: Command.Category) : DiabotCommand() {
         buildNightscoutResponse(endpoint, event)
     }
 
+    private fun processPebble(url: String, dto: NightscoutDTO, event: CommandEvent) {
+        val client = HttpClient()
+        val url_base = url.replace("/api/v1/", "/pebble")
+        val method = GetMethod(url_base)
+        val statusCode = client.executeMethod(method)
+
+        if (statusCode == -1) {
+            event.reactError()
+        }
+
+        val json = method.responseBodyAsStream.bufferedReader().use { it.readText() }
+
+        val jsonObject = JsonParser().parse(json).asJsonObject
+        val json2 = jsonObject.get("bgs").asJsonArray.get(0).asJsonObject
+
+        val cob = json2.get("cob").asInt
+        val iob = json2.get("iob").asFloat
+        val bgDelta = json2.get("bgdelta").asString
+        dto.iob = iob
+        dto.cob = cob
+        if (dto.delta == null) {
```

Comment:
is `dto.delta` ever not null?

Reply:
i believe it would be non-null if we got a delta value from the check of /api/v1/entries.json?count=1